scripts/digest_inputs

```diff
@@ -32,6 +32,19 @@ DEV_TAGS="${DEV_TAGS:-}"
 DOCKERFILE="${DOCKERFILE:-Dockerfile}"
 TAGS="${TAGS:-}"
 REDHAT_TAG="${REDHAT_TAG:-}"
+PUSH_AUTO_DEV_TAGS="${PUSH_AUTO_DEV_TAGS:=false}"
+
+# Strip version to MAJOR.MINOR
+MINOR="${VERSION%.*}"
+
+# Set default dev tags
+if [ "$PUSH_AUTO_DEV_TAGS" = true ] || ([[ ! -z "${DEV_TAGS}" ]]); then
+  DEV_TAGS=${DEV_TAGS:=("hashicorppreview/${REPO_NAME}:${MINOR}-dev" "hashicorppreview/${REPO_NAME}:${MINOR}-dev-${REVISION}")}
+fi
+
+# Check that one of tags and redhags is set
+[ -n "${TAGS:-}${REDHAT_TAG:-}" ] || die "Must set either TAGS or REDHAT_TAG"
+[ -z "${TAGS:-}" ] || [ -z "${REDHAT_TAG:-}" ] || die "Must set either TAGS or REDHAT_TAG (not both)"
```

Comment:
I think these 2 may be combined into one, right?

```
[ -n "${TAGS:-}" ] || [ -n "${REDHAT_TAG:-}" ] || die "Must set either TAGS or REDHAT_TAG (not both)"
```

Reply:
hmmm seemed to be a duplicate from above - must have been copy/paste error. deleting now